modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc

```diff
@@ -18,14 +18,16 @@ To connect to an FTP server, you will need:
 Your new mountpoint is available to all users by default, and you may restrict access by entering specific users or groups in the *Available for* field.
 
 Optionally, ownCloud can use FTPS (FTP over SSL) by checking `Secure ftps://`. 
-This requires additional configuration with your root certificate, if the FTP server uses https://en.wikipedia.org/wiki/Self-signed_certificate[a self-signed certificate].
+This requires additional configuration with your root certificate, if the FTP server uses
```

Comment:
Is there a specific need to split sentences over multiple lines?

Reply:
see above